Eatery/Views/Lookahead/FilterEateriesView.swift

```diff
@@ -20,6 +20,8 @@ class FilterEateriesView: UIView, UIGestureRecognizerDelegate {
 
     private let dateAndMealSeparator = UIView()
 
+    private(set) var separatorY: CGFloat = 0
```

Comment:
does this need to be `private(set)`? can we just make it private?

Reply:
We need to access it from `LookaheadTableViewController`